dev.lasheen.qr.yml

```diff
@@ -0,0 +1,28 @@
+app-id: dev.lasheen.qr
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: qr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: qr
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/YoussefLasheen/qr/releases/download/v1.0.0/qr-Linux-Portable.tar.gz
+        sha256: 8d4bd10e5ff14fba70e2fefff09b1fbf23d11dd53bfe2430e3173764bed6658d
+      - type: dir
```

Comment:
don't use `dir`. list the individual files.

Also the .desktop, appstream and icons should be shipped by upstream.